docs/tasks/administer-cluster/kubeadm-upgrade-1-8.md

```diff
@@ -195,52 +202,74 @@ $ kubeadm upgrade apply v1.8.0
    find your CNI provider and see if there are additional upgrade steps
    necessary.
 
-## Upgrading your worker nodes
+## Upgrading your master and nodes
 
-For each worker node (referred to as `$WORKER` below) in your cluster, upgrade `kubelet` by executing the following commands:
+For each host (referred to as `$HOST` below) in your cluster, upgrade `kubelet` by executing the following commands:
+
+1. Prepare the host for maintenance, marking it unschedulable and evicting the workload:
+
+```shell
+$ kubectl cordon $HOST
```

Comment:
Checked now that drain cordons as well, so this is not strictly needed, nit

Reply:
Done.